packages/babel-generator/test/generate-typescript-tests.js

```diff
@@ -0,0 +1,69 @@
+/*
+Copies tests from babylon's TypeScript test suite to babel-generator.
```

Comment:
kinda thinking that is another reason why we should move babylon back into the monorepo?